spec/rubocop/formatter/formatter_set_spec.rb

```diff
@@ -83,6 +83,16 @@ module Formatter
           formatter_set.add_formatter('simple')
         end
 
+        around do |example|
+          begin
+            stdout = $stdout
+            $stdout = StringIO.new
```

Comment:
We do something similar to this in several tests. The only difference is storing the original `$stdout` vs reseting it using `STDOUT`. Can this use `STDOUT`?

```ruby
begin
  $stdout = StringIO.new
  example.run
ensure
  $stdout = STDOUT
end
```